assets/app/scripts/directives/logViewer.js

```diff
@@ -275,6 +275,10 @@ angular.module('openshiftConsole')
                 $timeout(function() {
                   $scope.loading = false;
                 }, 100);
+
+                if(lastLineNumber === 0) {
+                    $scope.emptyState = true;
```

Comment:
Yeah, it works because the `$evalAsync` was trigging an eventual digest loop anyway. Better to pull this inside, though.